README.md

```diff
@@ -0,0 +1,4 @@
+This package enables Pimcore to write to an Azure Storage Account via Flysystem. It acts as a wrapper around https://github.com/thephpleague/flysystem-azure. To install:
+
+1. Run `composer require torqit/azure-flysystem-bundle:^1.0`
+2. Set environment variables and values for `AZURE_STORAGE_ACCOUNT_NAME`, `AZURE_STORAGE_ACCOUNT_CONTAINER` and `AZURE_STORAGE_ACCOUNT_KEY` that will allow the bundle to write to your Storage Account.
```

Comment:
3. Add `Local` environment variable to env, to only used the bundle in dev and prod
4. Add the following to `kernel.php` after install
`if (class_exists('TorqIT\\FlysystemAzureBundle\\FlysystemAzureBundle')) {
            if (!array_key_exists('LOCAL', $_ENV) || !$_ENV['LOCAL']) {
                $collection->addBundle(new FlysystemAzureBundle);
            }
        }`